test/ProteinStructureUtils_server_test.py

```diff
@@ -99,9 +171,29 @@ def test_model_upload(self):
             })[0]
         self.assertCountEqual(ret.keys(), ["structure_obj_ref", "report_ref", "report_name"])
 
+    #@unittest.skip('test_structure_to_pdb_file')
     def test_structure_to_pdb_file(self):
         ret = self.serviceImpl.structure_to_pdb_file(self.ctx, {'input_ref': self.pdb_ref,
                                                                 'destination_dir': self.scratch})
 
+    #@unittest.skip('test_export_structure')
     def test_export_structure(self):
         ret = self.serviceImpl.export_pdb(self.ctx, {'input_ref': self.pdb_ref})
+
+    def test_experiment_upload(self):
+        ret = self.serviceImpl.import_experiment_pdb_file(
+            self.ctx, {
+                'input_file_path': self.pdb_mmcif_file_path,
+                'structure_name': 'import_mmcif_test',
+                'workspace_name': self.wsName,
+            })[0]
+        self.assertCountEqual(ret.keys(), ["structure_obj_ref", "report_ref", "report_name"])
+
+    # @unittest.skip('test_structure_to_pdb_file')
+    def test_structure_to_mmcif_file(self):
+        ret = self.serviceImpl.structure_to_pdb_file(self.ctx, {'input_ref': self.pdb_mmCif_ref,
```

Comment:
Same here, you should exam the content of the created file.

Reply:
Yes.